quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-site/main.bicep

```diff
@@ -132,6 +135,7 @@ resource examplePacketCoreControlPlane 'Microsoft.MobileNetwork/packetCoreContro
         naptConfiguration: {
           enabled: naptEnabled
         }
+        dnsAddresses: dnsAddresses
```

Comment:
Almost certainly there is no way for bicep to disambiguate between the string 'dnsAddresses' and the parameter / array 'dnsAddresses', suggest you rename the parameter to something like 'dnsAddressesArray`., and that is why this is failing.

Reply:
Actually the issue is the lack of a parameter in https://github.com/Azure/azure-quickstart-templates/blob/dd49a4bb69637feb4c9603d2ca63f9a216b59d6f/quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-site/azuredeploy.parameters.json 

See https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=108413&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=326f581a-4b46-5a6e-a0f1-758ebdd2feec&l=28